docs/en/middleware/supplemental.md

```diff
@@ -489,15 +489,15 @@ import injector from '@dojo/framework/core/middleware/injector';
 ```
 
 -   `injector.subscribe(label: RegistryLabel, callback: Function = invalidator)`
-    -   Subscribes the given `callback` invalidation function against the specified registry `label` injector (if one exists). If a `callback` is not specified, the [`invalidator`](#invalidator) middleware is used by default so that the current widget will be marked as invalid and re-rendered when the injector makes its data available.
+    -   Subscribes the given `callback` invalidation function against the specified registry `label` injector (if one exists). If a `callback` is not specified, the [`invalidator`](/learn/middleware/core-render-middleware#invalidator) middleware is used by default so that the current widget will be marked as invalid and re-rendered when the injector makes its data available.
 -   `injector.get<T>(label: RegistryLabel): T | null`
     -   Retrieves the current injector associated with the given registry `label`, or `null` if no such injector exists.
 
 ## `block`
 
 Allows widgets to execute modules known as **blocks** within Node.js at build time. Typically used as part of build-time rendering.
 
-Described in detail in the [Building reference guide](../building/supplemental.md#build-time-rendering).
+Described in detail in the [Building reference guide](/learn/building/buildtime-rendering).
```

Comment:
Is this correct given how site is now generating URLs from headers? or should it be `build-time-rendering`?

Reply:
@sbinge yup it's correct, it renders as `buildtime-rendering`